extensions/entry-extension-collapsible/package.json

```diff
@@ -1,45 +1,55 @@
 {
   "name": "entry-extension-collapsible",
-  "version": "0.3.7",
+  "version": "0.1.0",
```

Comment:
Question: should I manually set the old version here, pretend to do a major/minor update to bump it to 1.0.0/0.4.0 or keep the new version (0.1.0)?

Reply:
I would bump it as it looks odd to go backwards 🤷🏾‍♂️